setup.sh

```diff
@@ -1,15 +1,14 @@
 #!/bin/sh -ex
 export PATH="/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
+export DEBIAN_FRONTEND=noninteractive
+export ETCKEEPER_NO_PUSH=1
+[ -f /.dockerenv ] && export DOCKER=true
 
-die() {
-    echo "$@" >&2
-    exit 1
-}
+apt-get install -y apt-transport-https git etckeeper
 
-# Some basic assumptions
-[ -d /etc-git/.git ]              || die "/etc-git is not a git repo, something has gone wrong"
-[ -f /usr/lib/apt/methods/https ] || die "Please install apt-transport-https"
-command -v etckeeper >/dev/null   || die "Please install etckeeper"
+if [[ ! -d "/etc-git" ]]; then
```

Comment:
`[[` isn't a POSIX SH thing, so the script cannot possibly run on Debian (whose `sh` is `dash` by default)